src/background.js

```diff
@@ -1101,6 +1101,8 @@ function initializeDispatcher() {
 		button.update();
 		utils.flushChromeMemoryCache();
 		cliqz.modules.core.action('refreshAppState');
+		// if user has whitelisted site, suspect broken page
+		metrics.handleBrokenPageTrigger(globals.BROKEN_PAGE_WHITELIST);
```

Comment:
@wlycdgr seems like this would be triggered when adding to and removing from the whitelist. Do we need both cases? If not we should check against `sitePolicy` to look for just the addition portion.

Reply:
Updated. Since `sitePolicy` is not currently being sent to the background and would have had to be added anyway, I instead send a `brokenPageMetricsWhitelistSite` bool to express intent more clealry and test for it in `PanelData#set`